### PR TITLE
Update folder start menu

### PR DESCRIPTION
**Pull request by ericpre**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ericpre** [comment]
See https://github.com/conda-forge/conda-forge.github.io/issues/1253 for reference.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ source:
     - py36-rtree-0.9.4.patch  # [win and py==36]
 
 build:
-  number: 0
+  number: 1
```
Comment (ccordoba12) [inline recipe/meta.yaml:18]: Please increase the build number to `2` (`1` was used in PR #110).

**Review by ccordoba12 — COMMENTED** [review]
Thanks for your help @ericpre. Looks good to me, except for the minor comment I left below.

File: recipe/meta.yaml (hunk 1 above)
Comment (ericpre) [reply recipe/meta.yaml]: Done

**Review by ccordoba12 — APPROVED** [review]
Thanks @ericpre!

**Comment by EBenkler** [comment]
@ccordoba12 and @ericpre Thank you for the help, updating menuinst in the base env from version 1.4.16 to 1.4.18 solved the issue.

**Comment by ccordoba12** [comment]
Miniconda doesn't come with menuinst.

**Pull request merged**